x-pack/plugins/file_data_visualizer/public/application/components/expanded_row/geo_point_content/format_utils.ts

```diff
@@ -8,7 +8,24 @@
 import { Feature, Point } from 'geojson';
 import { euiPaletteColorBlind } from '@elastic/eui';
 import { DEFAULT_GEO_REGEX } from './geo_point_content';
-import { SOURCE_TYPES } from '../../../../../../../../maps/common/constants';
+
+// TODO - copied from maps, fix import link, resolve circular dependency
```

Comment:
Can this be removed now that file_data_visualizer is its own plugin that depends on Maps and there is no longer a circular dependency?